falcon/routing/compiled.py

```diff
@@ -199,14 +199,25 @@ class can use suffixed responders to distinguish requests
         # NOTE(kgriffs): Fields may have whitespace in them, so sub
         # those before checking the rest of the URI template.
         if re.search(r'\s', _FIELD_PATTERN.sub('{FIELD}', uri_template)):
-            raise ValueError('URI templates may not include whitespace.')
+            raise UnacceptableRouteError('URI templates may not include whitespace.')
 
         path = uri_template.lstrip('/').split('/')
 
         used_names = set()
         for segment in path:
             self._validate_template_segment(segment, used_names)
 
+        def find_cp_converter(node):
+            value = [
+                (field, converter)
+                for field, converter, _ in node.var_converter_map
+                if self._converter_map[converter].CONSUME_MULTIPLE_SEGMENTS
```

Comment:
Do we really need to do that patching? We could just do a `getattr` here instead.

Reply:
no clue why I imokemented that way, but it seems we could avoid that

I think the idea was to have a place that could be used to normalize the api of the converters, in case in the future we wanted to do some other vhange to them.

but it does indeed seem overkill for this case